src/coreComponents/denseLinearAlgebra/interfaces/blaslapack/BlasLapackLA.cpp

```diff
@@ -855,4 +855,62 @@ void BlasLapackLA::matrixEigenvalues( MatRowMajor< real64 const > const & A,
   matrixEigenvalues( AT.toSliceConst(), lambda );
 }
 
+void BlasLapackLA::solveLinearSystem( MatColMajor< real64 const > const & A,
+                                      arraySlice1d< real64 const > const & rhs,
+                                      arraySlice1d< real64 > const & solution )
+{
+  // --- Check that source matrix is square
+  int const NN = LvArray::integerConversion< int >( A.size( 0 ));
+  GEOSX_ASSERT_MSG( NN > 0 &&
+                    NN == A.size( 1 ),
+                    "Matrix must be square" );
+
+  // --- Check that rhs and solution have appropriate dimension
+  GEOSX_ASSERT_MSG( rhs.size( 0 ) == NN,
+                    "right-hand-side vector has wrong dimensions" );
+
+  GEOSX_ASSERT_MSG( solution.size( 0 ) == NN,
+                    "solution vector has wrong dimensions" );
+
+  array1d< int > IPIV;
+  IPIV.resize( NN );
+  int const NRHS = 1; // we only allow for 1 rhs vector.
+  int INFO;
```

Comment:
You should respect our naming conventions too 😉

Reply:
for local variables you mean? We should change the entire file though because I think I used the convention used in this file when I added this function.

Reply:
Yes, I meant `iPiv`, `numRhs`, `info`.